### PR TITLE
Added keywords to everyday things category

### PR DESCRIPTION
**Pull request by robynk**
Changed transport, storage lockers & help with pets.

### CODE REVIEW

File: src/pages/personalisation/EverydayThingsSubcategories.js [hunk 1]
```diff
@@ -12,16 +12,16 @@ export default class EverydayThingsSubcategories extends BaseMultiQuestion {
         answers: {
             /* eslint-disable max-len */
             "Food packages/parcels/vouchers": append("food parcel voucher pack"),
-            "Transport": append("transport"),
+            "Transport": append("transport voucher taxi subsidy"),
```
Comment (DanielHeath) [inline src/pages/personalisation/EverydayThingsSubcategories.js:15]: Will this return anything matching 'subsidy'? Do we want `'"transport voucher" "taxi subsidy"'`?

File: src/pages/personalisation/EverydayThingsSubcategories.js [hunk 2]
```diff
@@ -12,16 +12,16 @@ export default class EverydayThingsSubcategories extends BaseMultiQuestion {
...
             "Keeping warm": append("swags blankets"),
             "Clothes": append("clothes"),
             "Showers": append("showers"),
             "Personal products": append("toiletries sanitary products"),
             "Laundry": append("laundry facilities washing drying"),
             "Household goods": append("household goods"),
-            "Storage lockers": append("storage lockers"),
+            "Storage lockers": remove("material aid").append("storage lockers"),
```
Comment (DanielHeath) [inline src/pages/personalisation/EverydayThingsSubcategories.js:22]: Similarly, do these need quoting?

File: src/pages/personalisation/EverydayThingsSubcategories.js (hunk 1 above)
Comment (robynk) [inline src/pages/personalisation/EverydayThingsSubcategories.js:15]: These seemed to return better results without quotes.

File: src/pages/personalisation/EverydayThingsSubcategories.js (hunk 2 above)
Comment (robynk) [inline src/pages/personalisation/EverydayThingsSubcategories.js:22]: These work better without quotes as well.

**Comment by kane-c** [comment]
:+1:

**Pull request merged**